### PR TITLE
Add a bunch of clothes

### PR DESCRIPTION
**Pull request by Firefox13**
9

## About The Pull Request

This PR adds clothing, and, to be precise:
Punk vest to loadout - only covers chest but gives some armor and pockets. Look epic.
изображение
NorthTech(NT for short) security uniform crate. Comes with an outdated vest that provides exceptional(60!) resist to energy weapons.
изображение
Inspector locker contains his usual inspector trenchcoat with fedora, as well as outdated brown trenchcoat and grey fedora... This PR updates sprites of those two.
изображение

## Why It's Good For The Game

poggers

## Changelog
:cl: Armolitskiy, Firefox13
add: Punk vest.
add: NT security uniform.
tweak: Old inspector trenchcoat and fedora are properly black now.
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
poggers

delegate stamp needs code review thank u

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -243,3 +243,9 @@ obj/item/clothing/under/genericr
 	desc = "A simple red shirt with black pants."
 	icon_state = "genericr"
 	item_state = "genericr"
+
+/obj/item/clothing/under/northtech
+	name = "NorthTech Security Uniform"
+	desc = "An outdated uniform of now extinct security corporation \"NewTech\". It was one of the first corporations to adopt and employ the use of energy weapons technology."
```
Comment (Wouju) [inline code/modules/clothing/under/miscellaneous.dm:249]: `name = "NorthTech Security Uniform"`
`desc = "An outdated uniform of now extinct security corporation \"NewTech\".`
Bruh moment, i assume this isn't intended?

**Review by Wouju — CHANGES_REQUESTED** [review]

**Comment by ArmolitskiyCarpoeb** [comment]
sprites by me

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -243,3 +243,9 @@ obj/item/clothing/under/genericr
 	desc = "A simple red shirt with black pants."
 	icon_state = "genericr"
 	item_state = "genericr"
+
+/obj/item/clothing/under/northtech
+	name = "NorthTech Security Uniform"
+	desc = "An outdated uniform of now extinct security corporation \"NorthTech\". It was one of the first corporations to adopt and employ the use of energy weapons technology."
```
Comment (Hopekz) [inline code/modules/clothing/under/miscellaneous.dm:249]:
```suggestion
	desc = "An outdated uniform of now-extinct security corporation \"NorthTech\". It was one of the first corporations to adopt and employ the use of energy weapons technology."
```

**Review by Hopekz — CHANGES_REQUESTED** [review]
resolve conflict for merge

**Comment by genessee596** [comment]
fix conflix brough

**Comment by Firefox13** [comment]
fuck this shit

**Pull request closed without merging**